app/models/business_contact_form.rb

```diff
@@ -0,0 +1,21 @@
+class BusinessContactForm < MailForm::Base
+  include ActiveModel::Validations
+
+  attribute :area_of_interest
+  attribute :company_name
+  attribute :email, validate: true
+  attribute :file, attachment: true
```

Comment:
The "file" name itself in this model is not descriptive. Let's give more information.
```suggestion
  attribute :file_in_attachment, attachment: true
```
or
```suggestion
  attribute :attached_file, attachment: true
```